SSO update (#828)
apollographql/docs: src/content/graphos/org/sso/okta-integration-guide.mdx

   - `given_name`: `user.firstName`
   - `family_name`: `user.lastName`

   <img
     src="../../img/sso/okta-attribute-statements.jpg"
     alt="Okta configure attributes statements"
     className="screenshot"
   />

   Then click **Next**.

8. In the **Feedback** step, select **I'm an Okta customer adding an internal app**, then click **Finish**.

### Step 2. Send SAML metadata to Apollo

1. From your new Okta SAML integration's details page, scroll down and click **View SAML setup instructions** on the right side:

   <img
     src="../../img/sso/okta-settings.jpg"
     alt="Okta settings"
     className="screenshot"
   />

2. In the dialog that appears, copy and paste the contents of the **IDP metadata** text box into a text file:

   <img
     src="../../img/sso/okta-idp-metadata.jpg"
     alt="Okta IdP metadata"
     className="screenshot"
   />

3. Send the text file to your Apollo contact. They will complete your SSO setup.

Once your SSO configuration is live, organization members must re-login to GraphOS Studio. Once you've confirmed the new configuration works as expected, remove any legacy Apollo integrations in Okta if you have them.

## Assign users in Okta

> Whether you're using the official Okta integration or creating your own, you need to assign users to it so they can access GraphOS. You can assign individual users or groups by following these steps:

Reviewer: ```suggestion
You'll next need to assign users to it so they can access GraphOS. You can assign individual users or groups by following these steps:
```

Author: https://github.com/apollographql/docs/pull/828/commits/b20bb03e2c6f68c722366f9b2317077e91439b2b
